src/transformers/models/whisper/tokenization_whisper.py

```diff
@@ -496,7 +496,7 @@ def _normalize(self, text):
         normalizer = EnglishTextNormalizer(self.english_spelling_normalizer)
         return normalizer(text)
 
-    def _decode_with_timestamps(self, token_ids, skip_special_tokens=False, time_precision=0.02) -> str:
```

Comment:
No need to skip special tokens here -> we do this in the pre-process method once before decoding